syntax = "proto3";

package proto.math.v1;

service MathService {
  rpc Add(AddRequest) returns (AddResponse){}
}

message AddRequest{
  int32 x = 1;
  int32 y = 2;
}

message AddResponse{
  int32 res = 1;
}